specification/openapi.json

```diff
@@ -0,0 +1,261 @@
+          },
+          "defaultFee": {
+            "type": "number",
+            "description": "When the transfer fee is variable, but an app has not yet calculated it"
+          },
+          "qqPrefix": {
+            "type": "string",
+            "description": "QR code prefix for an address"
+          },
+          "status": {
+            "type": "string",
+            "enum": ["active", "disabled"],
+            "description": "Should the coin be processed"
+          },
+          "createCoin": {
+            "type": "boolean",
+            "description": "Should an app itself create the coin or only use the info for the blockchain"
+          },
+          "defaultVisibility": {
+            "type": "boolean",
+            "description": "To show a coin by default, or hide it"
+          },
+          "defaultOrdinalLevel": {
+            "type": "integer",
+            "description": "Default ordinal number in a wallet list. Coins with the same ordinal number are sorted alphabetically. Coins without an order are shown last, alphabetically"
+          },
+          "consensus": {
+            "type": "string",
+            "description": "Blockchain consensus type"
+          },
+          "blockTimeFixed": {
+            "type": "integer",
+            "description": "Fixed block time in ms"
+          },
+          "blockTimeAvg": {
+            "type": "integer",
+            "description": "Average block time in ms"
+          },
+          "txFetchInfo": {
+            "type": "object",
+            "properties": {
+              "newPendingInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Pending\" for new transactions"
+              },
+              "oldPendingInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Pending\" for old transactions"
+              },
+              "registeredInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Registered\""
+              },
+              "newPendingAttempts": {
+                "type": "integer",
+                "description": "Attempts to fetch Tx when its current status is \"Pending\" for new transactions"
+              },
+              "oldPendingAttempts": {
+                "type": "integer",
+                "description": "Attempts to fetch Tx when its current status is \"Pending\" for old transactions"
+              }
+            }
+          },
+          "txConsistencyMaxTime": {
+            "type": "integer",
+            "description": "Time in ms when difference between in-chat transfer and Tx timestamp considered as acceptable"
+          },
+          "nodes": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
```

Comment:
1. Add [alt_ip](https://github.com/Adamant-im/adamant-wallets/blob/master/assets/general/adamant/info.json#L27) and [hasIndex](https://github.com/Adamant-im/adamant-wallets/blob/master/assets/general/ethereum/info.json#L37). Both are optional
2. Set `url` as required param

Reply:
@martiliones what is `hasIndex` for? There is no info in the README.md

Reply:
Probably, indicates whether [ETH Indexer](https://github.com/Adamant-im/ETH-transactions-storage) installed